OpenRA.Game/FileSystem/Folder.cs

```diff
@@ -41,13 +42,17 @@ public IEnumerable<string> Contents
 		public Stream GetStream(string filename)
 		{
 			try { return File.OpenRead(Path.Combine(path, filename)); }
-			catch { return null; }
+			catch
+			{
+				Log.Write("debug", "File {0} not found in {1}.".F(filename, path));
```

Comment:
The default behaviour for maps is to try `GetStream` to search for map-specific overrides, then fall back to the main filesystem if that returns null.  Adding this logging means that unpackaged maps write a lot of junk to the debug.log.

Can you please drop this change?